Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -839,12 +839,22 @@
         "Icon": "/Assets/Helpers.png",
         "DocumentationUrl": "https://raw.githubusercontent.com/MicrosoftDocs/WindowsCommunityToolkitDocs/master/docs/behaviors/AutoSelectBehavior.md"
       },
+      {
+        "Name": "KeyDownTriggerBehavior",
+        "Type": "KeyDownTriggerBehaviorPage",
+        "Subcategory": "Systems",
+        "About": "Behavior to listen to a key press on a control and executes actions",
+        "CodeUrl": "https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.UI.Behaviors/Keyboard/KeyDownTriggerBehavior.cs",
+        "XamlCodeFile": "KeyDownTriggerBehaviorXaml.bind",
```

Comment:
Since removed the package page, you don't have a `Type` anymore, which also means you need the full path to the bind file:

```suggestion
        "Subcategory": "Systems",
        "About": "Behavior to listen to a key press on a control and executes actions",
        "CodeUrl": "https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.UI.Behaviors/Keyboard/KeyDownTriggerBehavior.cs",
        "XamlCodeFile": "/SamplePages/KeyDownTriggerBehavior/KeyDownTriggerBehaviorXaml.bind",
```